scripts/version.sh

```diff
@@ -23,6 +23,7 @@
 #############################################################################
 
 BASEDIR=$(dirname "$0")
+unset CDPATH
```

Comment:
I still do not get why `CDPATH` would cause `cd` to print out, but regardless the fix would be better imho redirecting cd's stdout and stderr to `/dev/null` it would be more general.

Reply:
Quoting my comment I've sent via email:

```
That was my first solution however there are lots of cd calls that would need to be annotated. IIRC there's one right in the same script. cd prints the chosen directory to stdout if it thinks the directory was chosen from CDPATH. e.g. if you set CDPATH to "/" then cd tmp will print /tmp if you use it anywhere in the system where there's no directory named tmp.
```